marklogic-data-hub/src/main/java/com/marklogic/hub/impl/DataHubImpl.java

```diff
@@ -273,13 +274,14 @@ public boolean isServerVersionValid(String versionString) {
                     alteredString = StringUtils.rightPad(alteredString, 4, "0");
                 }
                 int ver = Integer.parseInt(alteredString.substring(0, 4));
-                if (!isNightly && ver < 9050) {
+                //Future proofing against 9.10 since we have an ES incompatibility that needs resolving
+                if (!isNightly && (ver < 9070 || ver > 9095)) {
                     return false;
                 }
             }
             if (isNightly) {
                 String dateString = versionString.replaceAll("[^-]+-(\\d{4})(\\d{2})(\\d{2})", "$1-$2-$3");
-                Date minDate = new GregorianCalendar(2018, 4, 5).getTime();
+                Date minDate = new GregorianCalendar(2018, 11, 5).getTime();
                 Date date = new SimpleDateFormat("y-M-d").parse(dateString);
```

Comment:
This corresponds to 12/5/2018 's nightly(GregorianCalendar's month is from 0 to 11).  So this test has to be changed
public void testValidateServer9nightly() {
        assertTrue(dh.isServerVersionValid("9.0-20181110"));
}

Reply:
Resolved via https://github.com/marklogic/marklogic-data-hub/pull/2448/commits/9ae168e1eb0a3fde726f6dd18fcf8ef62d89d9c0